packages/php/email-editor/src/Engine/Patterns/class-patterns.php

```diff
@@ -33,6 +33,21 @@ private function register_block_pattern_categories(): void {
 				'label'       => _x( 'Email Contents', 'Block pattern category', 'woocommerce' ),
 				'description' => __( 'A collection of email content layouts.', 'woocommerce' ),
 			),
+			array(
+				'name'        => 'newsletter',
+				'label'       => _x( 'Newsletter', 'Block pattern category', 'woocommerce' ),
+				'description' => __( 'A collection of newsletter email layouts.', 'woocommerce' ),
+			),
+			array(
+				'name'        => 'welcome',
+				'label'       => _x( 'Welcome', 'Block pattern category', 'woocommerce' ),
+				'description' => __( 'A collection of welcome email layouts.', 'woocommerce' ),
+			),
```

Comment:
It appears that these pattern categories are primarily needed by MailPoet. Is there a specific reason for registering them in the email editor core instead of within the MailPoet plugin?